forem/forem
# Refactoring SidebarWidget and sidebarUser

## What type of PR is this? 


## Description

Refactoring https://github.com/forem/forem/blob/master/app/javascript/sidebar-widget/SidebarWidget.jsx and
https://github.com/forem/forem/blob/master/app/javascript/sidebar-widget/sidebarUser.jsx
into functional component and simplifying the code

## Related Tickets & Documents
Closes https://github.com/forem/forem/issues/10948

## QA Instructions, Screenshots, Recordings
1. Go to url http://localhost:3000/t/discuss
2. Scroll down to see suggested users on the right panel and it should work without any change

## Added tests?


## Added to documentation?
